llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -5180,7 +5180,9 @@ MachineInstr *CombinerHelper::buildUDivUsingMul(MachineInstr &MI) {
   LLT ScalarShiftAmtTy = ShiftAmtTy.getScalarType();
 
   unsigned KnownLeadingZeros =
-      KB ? KB->getKnownBits(LHS).countMinLeadingZeros() : 0;
+      (!MI.getFlag(MachineInstr::MIFlag::IsExact) && KB)
```

Comment:
This is not related to vectors and is not mentioned in the description. Separate patch?

Reply:
Fixed!